### ISSUE TITLE
Require SSH keys to become a lug.org.uk hosting user.

### ISSUE THREAD

**Issue by JonTheNiceGuy** [lugorguk/test-deploy#8]
My personal feeling is "yes".

**Comment by RalphCorderoy** [comment]
Yes. Let them supply public information over an insecure channel rather than us trying to ferry private information back to them.

**Comment by gwestwood** [comment]
Yes.  This would make it more secure by allowing us to disable authentication by methods other than SSH keys.
